akka-cluster-sharding/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -3,3 +3,16 @@ ProblemFilters.exclude[Problem]("akka.cluster.sharding.Shard.*")
 
 # #25191
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.sharding.ShardRegion.retryTask")
+
+# #27100 Productionize: GetShardRegionStats returns empty shard set on ask timeout
+# askAllShards, an internal function, was renamed and changed to query all or a subset of shards to try failures only
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.sharding.ShardRegion.askAllShards")
+# Added new field for failed shard queries to akka.cluster.sharding.ShardRegion#ShardRegionStats, and updated in proto
+ProblemFilters.exclude[MissingTypesProblem]("akka.cluster.sharding.ShardRegion$ShardRegionStats$")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.sharding.ShardRegion#ShardRegionStats.apply")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.sharding.ShardRegion#ShardRegionStats.copy")
```

Comment:
Seems odd, nothing does a copy, this is meant as a read only. Perhaps we should have done class vs case class.

Reply:
Still it is a public type so we'll need to add a bin compat copy. We avoid case classes for public types like these as they are easier to evolve but this was already a case class

Reply:
Added, thanks.